local.properties

```diff
@@ -0,0 +1,8 @@
+## This file must *NOT* be checked into Version Control Systems,
+# as it contains information specific to your local configuration.
+#
+# Location of the SDK. This is only used by Gradle.
+# For customization when using a Version Control System, please read the
+# header note.
+#Mon Mar 09 17:04:00 CET 2020
+sdk.dir=/Users/davide/Library/Android/sdk
```

Comment:
I think we can remove the `local.properties` file as it's pointing to your user's directory 😅

Reply:
uppps, sure I'll remove it